server/config.go

```diff
@@ -0,0 +1,222 @@
+package server
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	jaeger "github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+	"github.com/uber/jaeger-lib/metrics"
+	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-vitess.v0/mysql"
+)
+
+const (
```

Comment:
You don't need this, you can use https://github.com/jaegertracing/jaeger-client-go/blob/6cc524e2736308fb6da8f054483d00baee39b0a1/config/config_env.go#L49 to get configuration from environment variables.

Reply:
And then, with this configuration, you can create a new tracer: https://sourcegraph.com/github.com/jaegertracing/jaeger-client-go@6cc524e2736308fb6da8f054483d00baee39b0a1/-/blob/config/config.go#L148

Reply:
Yeah, I know, but it was added recently and it's still just on master. I hope after they release a new version we'll be able to get rid of some stuff and directly use heir `FromEnv` functions.

Reply:
Oh, ok, sorry for the noise. Can you open an issue to remind change that on the future?

Reply:
Wait, I've just checked that they released a new version `2.13` two days ago (with env config), so on Friday the latest version was 2.12 (the version which we are using right now).
So, shall we integrate a new version?